### DISCUSSION TITLE
February 15th 2024 - 9:30am PST

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#21042] [Community Call]
Agenda below

**Comment by n-bajaj** [comment]
Azure Automation product group would be sharing the new feature releases: Runtime environment and support for Azure CLI commands in PowerShell runbooks.

**Comment by floh96** [comment]
If not already answered, please answer https://github.com/orgs/PowerShell/discussions/21138

**Reply by SydneyhSmith** [reply]
Thanks, we will plan to talk about this 😄

**Comment by SydneyhSmith** [comment]
## Agenda

- PowerShell 7.5 - Aditya
- Pseudo-terminal support - Patrick 
- Platform support - Jason
- Bug fixes and community PRs - Sydney
- PSGallery Fundamentals work - Aditya
- New types of repositories for PSResourceGet - Anam
- Concurrent installs - Amber
- Local caching of artifact details - Amber
- Intelligence in the shell - Steven
- Configuration - Michael
- Win32_OpenSSH - Danny
- SSHDConfig - Tess 
- Help system - Jason
- Welcome Justin 
- Docs update - Sean
- Working Group membership form - Jason

**Reply by Nova-Logic** [reply]
Would it be possible to discuss the https://github.com/PowerShell/PowerShell/issues/21114 . That is related system.management.automation, PowerShell and win32_openssh. It would be great if we would be able to create some mechanism that would make a management of windows hosts from linux easier by creating a way to connect to windows host through C# from linux hosts and getting proper Ad tickets(network sessions).  Because at this moment SshConnectionInfo not allowing to use password based auth and also not allowing other ways to make it done(may it would be possible to modify SshConnectionInfo to support GSSAPI)

**Reply by StevenBucher98** [reply]
March Community call announcement

**Reply by StevenBucher98** [reply]
Intern intro

**Comment by ThomasNieto** [comment]
PSResourceGet v3 feeds lack dependency support: https://github.com/PowerShell/PSResourceGet/issues/1407
